tests/publiccloud/instance_overview.pm

```diff
@@ -52,7 +56,16 @@ sub run {
     upload_logs('/var/log/zypper.log');
 
     assert_script_run("SUSEConnect --status-text");
-    zypper_call("lr");
+    zypper_call("lr -d");
+}
+
+sub test_flags {
+    return {fatal => 1};
+}
+
+sub post_fail_hook {
+    # Ensure instance is cleaned up on failure
+    provider::cleanup();
```

Comment:
have you forgot to call the super::post_fail_hook here?

Reply:
I'm using now the approach suggested by @asmorodskyi to use `publiccloud::basetest` as the base for this test case.